src/arithmetic.jl

```diff
@@ -162,7 +162,10 @@ for T in (:Taylor1, :HomogeneousPolynomial, :TaylorN)
 end
 
 for T in (:Taylor1, :TaylorN)
-    @eval zero(a::$T) = $T(zero.(a.coeffs))
+    @eval function zero(a::$T)
+        return $T(zero.(a.coeffs))
+        return za
```

Comment:
I guess this is some leftover of some tests you previously made, right?

Reply:
Good catch, thanks! It's now fixed